Products/ZenModel/ZenossInfo.py

```diff
@@ -279,6 +281,7 @@ def getZopeVersion(self):
         major, minor, micro, status, release = version.getZopeVersion()
         return Version(name, major, minor, micro)
 
+    @FunctionCache('getZenossRevision')
```

Comment:
but anyway it looks reasonable for me to remove this method at all. Will make a new pull request.